package.json

```diff
@@ -78,10 +85,9 @@
         "prop-types": "^15.5.10",
         "querystring": "^0.2.0",
         "radium": "^0.25.1",
-        "react": "^16.0.0",
-        "react-addons-test-utils": "^15.0.1",
```

Comment:
It looks like you guys have already migrated to using `react-dom/test-utils` so this doesn't appear to be needed anymore